### ISSUE TITLE
🔴 [P0] Захардкоженные метрики: счётчик «11 моделей», GPU, форматы

### ISSUE THREAD

**Issue by OlegKarenkikh** [dispersi0no/CHAT-VLM-LLM#3]
## Описание

Все метрики на главной странице — статические строки, не отражающие реальность.

## Текущее поведение

```python
col1.metric("Модели", "11")           # Реально: 4-5 рабочих
col2.metric("Форматы", "5+")          # Должно считаться из config
col3.metric("GPU", "NVIDIA Blackwell")  # Должно определяться динамически
```

Также захардкожены тексты:
- `"✅ 11 моделей интегрировано (5 новых + 6 базовых)"`
- `"✅ 9 моделей полностью рабочих из 11 настроенных"`

## Ожидаемое поведение

```python
config = load_config()
total_models = len(config.get('vllm', {})) + len(config.get('transformers', {}))
col1.metric("Модели", str(total_models))

formats = config.get('ocr', {}).get('supported_formats', [])
col2.metric("Форматы", str(len(formats)))

try:
    import torch
    gpu_name = torch.cuda.get_device_name(0) if torch.cuda.is_available() else "CPU"
except:
    gpu_name = "CPU"
col3.metric("GPU", gpu_name)
```

## Анализ источников числа «11»

| Источник | Кол-во |
|----------|--------|
| `MODEL_REGISTRY` в model_loader.py | 7 ключей |
| `config.yaml → models:` | 1 (остальные закомментированы) |
| `config.yaml → transformers:` | 2 |
| `config.yaml → vllm:` | 3 |
| `SingleContainerManager` | 4 контейнера |

Реально рабочих: **4-5 моделей**.

## Файлы
- `app.py` — метрики на главной странице

## Labels
`bug`, `priority: critical`

**Comment by OlegKarenkikh** [comment]
✅ **Решено в PR #2**

Метрики на главной странице теперь динамические:
- `ui/pages/home.py` считает модели из `config` (`vllm_count + transformers_count`)
- Форматы берутся из `config.get('ocr', {}).get('supported_formats', [])`
- GPU определяется через `torch.cuda.get_device_name(0)`
- Тексты достижений также динамические: `f"{total_models} моделей интегрировано ({vllm_count} vLLM + {transformers_count} Transformers)"`
